src/main/java/org/codeforamerica/shiba/output/applicationinputsmappers/UtilityPaymentsInputsMapper.java

```diff
@@ -36,13 +38,18 @@
     return map(application.getApplicationData().getPagesData());
   }
 
-  private List<ApplicationInput> map(PagesData pagesData) {
+  @Override
+  protected OneToManyParams getParams() {
+    return new OneToManyParams("utilityPayments", UTILITY_PAYMENTS, UTILITY_PAYMENTS_OPTIONS);
+  }
+
+  protected List<ApplicationInput> map(PagesData pagesData) {
```

Comment:
I like that this list lives in this file now instead of OneToManyApplicationInputMapper!